encord/workflow/stages/annotation.py

```diff
@@ -40,6 +46,17 @@ def get_tasks(
         dataset_hash: Union[List[UUID], UUID, List[str], str, None] = None,
         data_title: Optional[str] = None,
     ) -> Iterable[AnnotationTask]:
+        """
+        **Params**
+
+        - name: Name of the stage.
+        - uuid: Unique identifier for the stage.
+        - type_: The type of stage.
+
+        **Returns
+
+        Returns Annotatoin Workflow stages from non-Consensus and Consensus Projects.
```

Comment:
Typo

Reply:
Updated

Reply:
I'm not sure if we should mention Consensus here. Also terminology seems to be a bit off, it returns tasks, not stages.

So perhaps just smth like:
`Returns Annotation tasks for the stage`
?

Reply:
Updated